### ISSUE TITLE
If the test-dataset also have some noise data, do you think it is hard to get the real performance?

### ISSUE THREAD

**Issue by guotong1988** [ZJU-DAILY/SGDL#4]
This question is based on https://github.com/ZJU-DAILY/SGDL/issues/3.
Thank you very much!

**Comment by CharlieHu99** [comment]
If test set is also noisy, its data distribution is consistent with the training set, which is not our major concern. It can be solved by other **normal** machine learning solutions. We are happy to answer questions related to our paper. Thanks!

**Comment by guotong1988** [comment]
Thank you again!
